src/bit-systems/media-loading.ts

```diff
@@ -19,6 +20,39 @@ import { loadAudio } from "../utils/load-audio";
 import { MediaType, mediaTypeName, resolveMediaInfo } from "../utils/media-utils";
 import { EntityID } from "../utils/networking-types";
 
+export const MediaContentBounds = (MediaLoaded as any).contentBounds as Map<EntityID, Vector3>;
+
+const getBox = (() => {
+  const rotation = new THREE.Euler();
+  return (world: HubsWorld, eid: EntityID, rootEid: EntityID, worldSpace?: boolean) => {
+    const box = new THREE.Box3();
+    const obj = world.eid2obj.get(eid)!;
+    const rootObj = world.eid2obj.get(rootEid)!;
+
+    rotation.copy(obj.rotation);
+    obj.rotation.set(0, 0, 0);
+    obj.updateMatrices(true, true);
+    rootObj.updateMatrices(true, true);
+    rootObj.updateMatrixWorld(true);
+
+    computeObjectAABB(rootObj, box, false);
+
+    if (!box.isEmpty()) {
+      if (!worldSpace) {
+        obj.worldToLocal(box.min);
+        obj.worldToLocal(box.max);
+      }
+      obj.rotation.copy(rotation);
+      obj.matrixNeedsUpdate = true;
+    }
+
+    rootObj.matrixWorldNeedsUpdate = true;
+    rootObj.updateMatrices();
+
+    return box;
+  };
+})();
```

Comment:
Not a blocking issue, but a comment for future clean up.

I think `media-loading` system is one of the most complex systems and it may be hard to be maintained. And now we are adding a new code.

Perhaps the reasons of the complexity is `media-loading` system processes multiple things, like resolving URL, loading media, animating object after loading. If we can break `media-loading` system up to multiple systems, they can be simpler. And if we do that, this media content bound setting would be move to either one, or a new system.